UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.API.FunctionalTests/Helper/AuthTokenProvider.cs

```diff
@@ -66,6 +116,51 @@ private static async Task<string> GenerateEssToken(string ClientId, string Clien
             }
 	     return Token;
        }
+        public async Task<string> GetScsToken()
+        {
+            ScsAccessToken = await GenerateScsToken(ScsAuthConfig.AutoTestClientId, ScsAuthConfig.AutoTestClientSecret, ScsAccessToken);
+            return ScsAccessToken;
+        }
+
+        /// <summary>
+        /// Generate SCS Token
+        /// </summary>
+        /// <param name="ClientId"></param>
+        /// <param name="ClientSecret"></param>
+        /// <param name="Token"></param>
+        /// <returns></returns>
+
+        private static async Task<string> GenerateScsToken(string ClientId, string ClientSecret, string Token)
+        {
+            string[] scopes = new string[] { $"{ScsAuthConfig.ScsClientId}/user_impersonation" };
+            if (Token == null)
+            {
+                if (EssauthConfig.IsRunningOnLocalMachine)
```

Comment:
Kindly change this to if (ScsAuthConfig.IsRunningOnLocalMachine)

Reply:
Resolved